README.md

```diff
@@ -390,15 +396,15 @@ Example response:
 * Authorized users can generate API keys for access.
 * Awardees are responsible for rotating keys on a three day timeframe.
     ** Permissions will be revoked after this time.
-* Awardee must specify Awardee or Organization/Site of Awardee in call to API.
+* Awardee must specify Awardee  in call to API.
```

Comment:
I would rephrase this as:

Service accounts for specific awardees must specify the "awardee" parameter in their requests.

Reply:
done